protons/integrators.py

```diff
@@ -0,0 +1,146 @@
+import numpy
+import simtk.unit
+import simtk.unit as units
+import simtk.openmm as mm
+kB = units.BOLTZMANN_CONSTANT_kB * units.AVOGADRO_CONSTANT_NA
+
+class GHMCIntegrator(mm.CustomIntegrator):
```

Comment:
Was this taken from any repo in particular, or written for this specific case? We may want to make a note of the original author in that case to keep track of contributors.